src/main/java/edu/kis/powp/jobs2d/features/MouseFeature.java

```diff
@@ -0,0 +1,62 @@
+package edu.kis.powp.jobs2d.features;
+
+import javax.swing.JPanel;
+import javax.swing.event.MouseInputListener;
+
+import edu.kis.powp.appbase.Application;
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.DriverManager;
+
+public class MouseFeature {
```

Comment:
It does not look similarly to other "features" classes in application. I think that more specific name would be more appropriate.